scripts/gpu_support/nvidia/install_cuda_and_libraries.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
+
+# This script can be used to install CUDA and other libraries by NVIDIA under
+# the `.../host_injections` directory.
+#
+# This provides the parts of the CUDA installation and other libriaries that
+# cannot be redistributed as part of EESSI due to license limitations. While
+# GPU-based software from EESSI will _run_ without these, installation of
+# additional software that builds upon CUDA or other libraries requires that
+# these installation are present under `host_injections`.
+#
+# The `host_injections` directory is a variant symlink that by default points to
+# `/opt/eessi`, unless otherwise defined in the local CVMFS configuration (see
+# https://cvmfs.readthedocs.io/en/stable/cpt-repo.html#variant-symlinks). For the
+# installation to be successful, this directory needs to be writeable by the user
+# executing this script.
+
+# Initialise our bash functions
+TOPDIR=$(dirname $(realpath $BASH_SOURCE))
+source "$TOPDIR"/../../utils.sh
+
+# Function to display help message
+show_help() {
+    echo "Usage: $0 [OPTIONS]"
+    echo "Options:"
+    echo "  --help                           Display this help message"
+    echo "  --accept-cuda-eula               You _must_ accept the CUDA EULA to install"
```

Comment:
cuDNN also has a EULA (I think it is even the same EULA), this too needs to be accepted. The cuDNN easyblock currently does not support this but should (similar to what was done in https://github.com/easybuilders/easybuild-easyblocks/pull/3045)

Reply:
That will force this option to be more flexible (or we explicitly add an option for every package...not necessarily a bad option)

Reply:
I've added the option. Still todo: add an --include-easyblocks-from-commit for cuDNN in the easystack file

Reply:
Added to the EasyStack file